cmd/heznode/main.go

```diff
@@ -433,6 +437,138 @@ func cmdDiscard(c *cli.Context) error {
 	return nil
 }
 
+func cmdMakeBackup(c *cli.Context) error {
+	log.Info("starting to make a hermez-node backup...")
+	var wg sync.WaitGroup
+	// two goroutines for state db and postgres db
+	goroutinesAmount := 2
+	wg.Add(goroutinesAmount)
+	_cfg, err := parseCli(c)
+	if err != nil {
+		return tracerr.Wrap(fmt.Errorf("error parsing flags and config: %w", err))
+	}
+	cfg := _cfg.node
+	zipPath := c.String(flagPath)
+	if _, err = os.Stat(zipPath); os.IsNotExist(err) {
+		log.Infof("creating directory %s for the backup", zipPath)
+		err = os.MkdirAll(zipPath, os.ModePerm)
+		if err != nil {
+			log.Errorf("failed to create %s directory, err: %v", zipPath, err)
+			return err
+		}
+	}
+
+	backupPath := path.Join(zipPath, "backup")
+	err = os.MkdirAll(backupPath, os.ModePerm)
+	if err != nil {
+		log.Errorf("failed to create %s directory, err: %v", backupPath, err)
+		return err
+	}
+
+	today := time.Now().Format("20060102150405")
+	go makeDBDump(&wg, cfg, backupPath, today)
+	go makeStateDBDump(&wg, cfg, backupPath)
+
+	wg.Wait()
+	log.Info("finished with making state db dump")
+	log.Info("finished with dumps. started to make zip file...")
+	cmdZip := "zip"
+	args := []string{"-r", path.Join(zipPath, fmt.Sprintf("hermez-%s.zip", today)), backupPath}
```

Comment:
I got this error: ```2021-08-17T09:51:29Z	ERROR	heznode/main.go:479	failed to zip /home/ubuntu/src/backup directory, err: exec: "zip": executable file not found in $PATH

Error: exec: "zip": executable file not found in $PATH``` 

I believe it would be great don't demand to the user to have external zip library as Go has standard libraries to generate zip files. Please change it to use Go libs.